packages/trader/src/Modules/Reports/Components/filter-component.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Dropdown } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+import CompositeCalendar from 'App/Components/Form/CompositeCalendar/composite-calendar.jsx';
+
+const TransactionFilter = ({ handleFilterChange }) => {
+    const [default_filter, setDefaultFilter] = React.useState('all');
+
+    const filter_list = [
+        {
+            text: localize('All transactions'),
+            value: 'all',
+        },
+        {
+            text: localize('Buy'),
+            value: 'buy',
+        },
+        {
+            text: localize('Sell'),
+            value: 'sell',
+        },
+        {
+            text: localize('Deposit'),
+            value: 'deposit',
+        },
+        {
+            text: localize('Withdrawal'),
+            value: 'withdrawal',
+        },
+    ];
+
+    return (
+        <Dropdown
+            list={filter_list}
+            value={default_filter}
+            name='transaction-filter-dropdown'
+            className='dropdown-statement-filter'
+            classNameDisplay='dc-dropdown__display--suffix-icon'
+            has_symbol={false}
```

Comment:
You can omit this if it's `false`.

Reply:
when I omit the `false` it will not show the displaynames of the `dropdown`, I saw on other file, it still use the `false` argument